bitseq/store.go

```diff
@@ -70,6 +70,11 @@ func (h *Handle) Exists() bool {
 	return h.dbExists
 }
 
+// DataScope method returns the storage scope of the datastore
+func (h *Handle) DataScope() datastore.DataScope {
+	return datastore.GlobalScope
```

Comment:
When asked about this I said bitseq ds should be global scope, but actually it makes more sense this property is configurable by who instantiates bitseq. We can have an local or global scoped id manager.

As @mavenugo said this is not a concern of this PR. It's an incremental change after this PR is merged.